components/ScrollingAvatars/ScrollingAvatarsContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, useState } from 'react';
+import _ from 'underscore';
+
+import ScrollingAvatars from './ScrollingAvatars';
+
+interface AvatarInfo {
+  name: string;
+  position: string;
+  imgSrc: string;
+  key?: number;
+}
+
+interface IProps {
+  rows?: number;
+  startScrollRight?: boolean;
+  avatarInfos: AvatarInfo[];
+}
+
+const ScrollingAvatarsContainer: FC<IProps> = ({ rows = 1, avatarInfos, startScrollRight }) => {
+  const [name, setName] = useState<string>('');
+  const [position, setPosition] = useState<string>('');
+  const titleString = name && position ? `${name} - ${position}` : '';
+  let mappedScrollingAvatars;
+
+  if (rows >= avatarInfos.length) {
+    mappedScrollingAvatars = Array.from(Array(rows).keys()).map((key) => (
+      <ScrollingAvatars
+        key={key}
+        scrollDirection={key % 2 === 0 && !startScrollRight ? 'left' : 'right'}
+        avatarInfos={avatarInfos}
+        setName={setName}
+        setPosition={setPosition}
+      />
+    ));
+  } else {
+    let currKey = 0;
+    mappedScrollingAvatars = _.chunk(avatarInfos, avatarInfos.length / rows).map(
```

Comment:
I get what's going on here, but I'd like some documentation in this function body in general. It's a bit hard to follow, especially compared to the rest of the code in this PR